roadside-forms-frontend/frontend_web_app/src/components/Event/validationSchema.js

```diff
@@ -135,22 +105,44 @@ export const validationSchema = Yup.object().shape({
       // Return true if the field is empty
       return true;
     }
+
     const currentDate = new Date();
     const inputDate = new Date(dob);
 
+    // Set the time zone offset to Pacific Standard Time (PST)
+    const pacificOffset = 480; // PST offset is 480 minutes (8 hours)
+    inputDate.setMinutes(inputDate.getMinutes() + pacificOffset);
```

Comment:
are we converting from UTC to PST or PST to UTC here?

Reply:
oh i see the issue with this logic, this is not going to work with everytime zone convert, I guess I will need to use some liberary to handle converting the timezone to PST.